### PR TITLE
feat: new project card

### PR DESCRIPTION
**Pull request by Tymek**
## About the changes
image

- [x] Added "updated at"
- [x] Redone flags/health/members
- [x] Moved "favorite" action
- [x] New project mode badge
- [x] New "last active" item

### CODE REVIEW

File: frontend/src/interfaces/project.ts [hunk 1]
```diff
@@ -7,13 +7,16 @@ export interface IProjectCard {
     name: string;
     id: string;
     createdAt: string;
-    health: number;
```
Comment (kwasniew) [inline frontend/src/interfaces/project.ts:10]: what has changes that those are now optional? was it wrong before?

File: frontend/src/component/project/ProjectCard/ProjectCardFooter/ProjectOwners/ProjectOwners.tsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+const StyledOwnerName = styled('div')(({ theme }) => ({
+    display: 'flex',
+    flexDirection: 'column',
+    gap: theme.spacing(0.25),
+    margin: theme.spacing(0, 0, 0, 1),
+}));
+
+const StyledHeader = styled('span')(({ theme }) => ({
+    // margin: theme.spacing(0, 0, 0, 1),
```
Comment (kwasniew) [inline frontend/src/component/project/ProjectCard/ProjectCardFooter/ProjectOwners/ProjectOwners.tsx:73]: leftover comment?

File: frontend/src/component/project/ProjectCard/ProjectCardFooter/ProjectOwners/ProjectOwners.tsx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+import {
+    AvatarGroup,
+    AvatarComponent,
+} from 'component/common/AvatarGroup/AvatarGroup';
+
+export interface IProjectOwnersProps {
+    owners?: ProjectSchema['owners'];
+}
+
+const useOwnersMap = () => {
```
Comment (kwasniew) [inline frontend/src/component/project/ProjectCard/ProjectCardFooter/ProjectOwners/ProjectOwners.tsx:15]: probably needs a test in a separate PR

**Review by kwasniew — APPROVED** [review]
I'd also improve the state of loading images/gravatars. There's a short period when it shows a broken image

File: frontend/src/interfaces/project.ts (hunk 1 above)
Comment (Tymek) [reply frontend/src/interfaces/project.ts:10]: Now it supports also archive, and it's easier to solve it this way then with generic.

File: frontend/src/component/project/ProjectCard/ProjectCardFooter/ProjectOwners/ProjectOwners.tsx (hunk 2 above)
Comment (Tymek) [inline frontend/src/component/project/ProjectCard/ProjectCardFooter/ProjectOwners/ProjectOwners.tsx:73]:
```suggestion
```

**Pull request merged**